internal/common/common.go

```diff
@@ -71,6 +71,9 @@ func AllStrings(vs []string, f func(string) bool) bool {
 // GetBootstrapHost return host that was set as bootstrap
 func GetBootstrapHost(cluster *Cluster) *models.Host {
 	for _, host := range cluster.Hosts {
+		if IsSingleNodeCluster(cluster) && cluster.APIVip == "" {
```

Comment:
why we validate  cluster.APIVip == "" ? maybe this one should be validated before you start running this flow?

Reply:
if we will move to installing state we will not need that check.

Reply:
You are absolutely right. When adding the code fix in `InstallCluster`, I don't need to make any changes into `GetBootstrapHost`